src/Cli/AzureCliLogin.ts

```diff
@@ -7,14 +7,20 @@ import * as io from '@actions/io';
 export class AzureCliLogin {
     loginConfig: LoginConfig;
     azPath: string;
-    
+    loginOptions: ExecOptions;

```

Comment:
I am not seeing `loginOptions` being used when instantiating `AzureCliLogin`. Why is it put in the `constructor`?